### DISCUSSION TITLE
Show Orchard now runs on Orchard Core

### DISCUSSION THREAD

**Discussion by Piedone** [OrchardCMS/OrchardCore#14525] [Show and tell]
We just migrated [Show Orchard](https://showorchard.com/), the gallery of exemplary Orchard Core and Orchard 1.x sites from Orchard 1 to Orchard Core, still running on DotNest. Check out our case study here: https://dotnest.com/blog/show-orchard-case-study-migrating-an-orchard-1-dotnest-site-to-orchard-core

**Comment by hishamco** [comment]
Woohoo, finally you made it

**Reply by hishamco** [reply]
Seems you didn't update https://github.com/Lombiq/Show-Orchard-Theme yet, right?

**Reply by Piedone** [reply]
Yeah, that's the old theme. The new one is in a private solution. But we could actually make it open-source.

**Reply by hishamco** [reply]
I see, I thought it was open source, so you accept the community contribution if it's possible

Greate job as usual

**Reply by Piedone** [reply]
Yeah, good point, we'll look into it. Thank you!

**Comment by wAsnk** [comment]
We just released the new OC theme as open source. You can find it here https://github.com/Lombiq/Show-Orchard-Theme in the dev branch.

The O1 theme can be found in the same repo, on branch `dev-orchard-1`.
